hugo/content/docs/dependency_injection.md

```diff
@@ -63,6 +63,29 @@ configureModelElement(context, 'task', SNode, TaskNodeView, {
 ```
 the `configureModelElement` method takes as an optional last parameter an object containing arrays for `enabled` and `disabled` features containing Symbols representing those features. Through this, we can disable default functionality like dragging or selecting nodes and add functionality, either custom or loaded through other non-default modules.
 
+## Creating custom components
+As described previously Sprotty uses inversifyJs for dependency injection. That means when creating own feature, views, etc. we have to use it too.
+As an example lets look at Sprottys `PolylineEdgeView`.
+
+```Typescript
+@injectable()
+export class PolylineEdgeView extends RoutableView {
+
+    @inject(EdgeRouterRegistry) edgeRouterRegistry: EdgeRouterRegistry;
+    ...
+}
+```
+The most important thing for our component to available in Sprotty is annotating it with `@injectable()`. Otherwise the dependency injection won't work.
```

Comment:
```suggestion
The most important thing for our component to be available in Sprotty is annotating it with `@injectable()`. Otherwise the dependency injection won't work.
```